planet4-gutenberg-blocks.php

```diff
@@ -234,11 +234,9 @@ function set_allowed_block_types( $allowed_block_types, $post ) {
 		// --- Add here your own Block Controller ---
 		// DEPRECATED: Blocks could be registered inside Loader class
 		// 'P4GBKS\Controllers\Blocks\NewCovers_Controller'
-		'P4GBKS\Controllers\Menu\Settings_Controller',
-		'P4GBKS\Controllers\Menu\Blocks_Usage_Controller',
-		'P4GBKS\Controllers\Menu\Reusable_Blocks_Controller',
+		\P4GBKS\Controllers\Menu\Settings_Controller::class,
+		\P4GBKS\Controllers\Menu\Blocks_Usage_Controller::class,
+		\P4GBKS\Controllers\Menu\Reusable_Blocks_Controller::class,
 	],
-	'P4GBKS\Views\View',
-	'P4GBKS\Command\Shortcode_To_Gutenberg',
-	'P4GBKS\Command\Controller'
+	\P4GBKS\Views\View::class
```

Comment:
The lines below this, being parameter 3 and 4 of the function call, had no effect as the function only has 2 parameters